library/Zend/ServiceMananger/Proxy/LazyServiceFactory.php

```diff
@@ -22,17 +21,17 @@
  *
  * @link https://github.com/Ocramius/ProxyManager/blob/master/docs/lazy-loading-value-holder.md
  */
-class LazyServiceFactory implements DelegatorFactoryInterface
+final class LazyServiceFactory implements DelegatorFactoryInterface
```

Comment:
Why are you making this final and properties private? Is there really a reason why this class must never be extended and (if it wasn't final) derived classes must never manipulate the properties?